.evergreen/csfle/set-temp-creds.sh

```diff
@@ -0,0 +1,30 @@
+# Obtains temporary AWS credentials for CSFLE testing.
+#
+# Run with a . to add environment variables to the current shell:
+# . ./set-temp-creds.sh
+#
+# Requires the python AWS SDK boto3. This can be installed with: pip install boto3
+# The path to python in a virtual environment may be passed with the PYTHON
+# environment variable.
+#
+# Requires AWS credentials for CSFLE to obtain temporary credentials.
+# Those credentials can be passed through the following environment variables:
+#
+# export AWS_ACCESS_KEY_ID=...
```

Comment:
Is there a reason we don't want to just export the credentials with the env var names that actually are used by the driver?

Reply:
Perhaps I am misunderstanding the question. CSFLE credentials cannot be configured with environment variables, so the output variable names I chose are arbitrary. The input variable names are consistent with MONGODB-AWS auth (with the addition of AWS_DEFAULT_REGION) to pass along to boto3.

Reply:
The comment here says that these credentials need to be passed through to `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, etc. to be used. I'm wondering why we don't just set those variables directly from within this script rather than requiring users to manually set them from the variables that we set instead in this script.

Reply:
Based on offline discussion, I misunderstood what this comment was saying and thought that the variables set by this script needed to be set to `AWS_ACCESS_KEY_ID` and the others mentioned. Given that this is not the case, I don't think any changes need to be made from where the script was before my previous comment; re-exporting the variables from within the script only will do something if they're already set, which means nothing will actually be changed in that case.

Reply:
Reverted!